relays/ethereum/src/ethereum_client.rs

```diff
@@ -396,8 +401,12 @@ async fn submit_ethereum_transaction(
 		estimate_gas(
 			client,
 			CallRequest {
-				to: contract_address,
+				to: contract_address.unwrap_or_default(),
```

Comment:
When creating a contract, an address of `None` was used. I'm not sure how to represent that now since the `to` field isn't an `Option` anymore.

@svyatonik, what do you think we should have here?

Reply:
This is not really correct. Deploying a contract must not provide `to` field at all, with current code it will test a call to `0x0` contract. Maybe it's an issue with `CallRequest` type from `web3` crate?

Reply:
In eth client it is `Option` - so probably yes, it could be changed to `Option` in rust-web3 as well. There's also `TransactionRequest`, which has almost same fields, but is used in different methods (`eth_sendTransaction`).